presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -400,6 +401,27 @@ protected Scope visitDelete(Delete node, Optional<Scope> scope)
             return createAndAssignScope(node, scope, Field.newUnqualified("rows", BIGINT));
         }
 
+        @Override
+        protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
```

Comment:
are you going to update access controls also, so we could check if user is privileged to analyze the table?

Reply:
Will add it later.

Reply:
Because one may want to disable that for certain users.